verilog/parser/verilog.lex

```diff
@@ -920,6 +925,21 @@ zi_zp { UpdateLocation(); return TK_zi_zp; }
   return TK_OTHER;
 }
 
+<PP_EXPECT_INCLUDE_FILE>{
+  {StringLiteral} {
```

Comment:
Just as note, as an alternative, you could enter different string sub-states to accumulate string contents (`yymore`), as opposed to using a single expression.  The advantage of doing so is that it can give better diagnostics, such as "Unterminated string".  This could be follow-up work that includes lex-fail tests.

Reply:
Right now, I am just re-using the already existing `StringLiteral`.
For now, I added a TODO where it is defined for future consideration.